libseccomp-sys/src/lib.rs

```diff
@@ -252,6 +258,8 @@ pub const SCMP_ACT_LOG: u32 = 0x7ffc0000;
 /// Allow the syscall to be executed
 pub const SCMP_ACT_ALLOW: u32 = 0x7fff0000;
 
+pub type size_t = c_ulong;
```

Comment:
Suggestions to name it like https://doc.rust-lang.org/core/ffi/type.c_size_t.html which can be used once it is stable.

Reply:
I decided to use `usize` instead of `c_ulong` because `size_t` of C is almost equivalent to `usize` of Rust.
But, rust-bindgen community has been discussed about it for long time.
https://github.com/rust-lang/rust-bindgen/issues/1671

Reply:
Yes. `usize` should be the same as `(c_)size_t` on all your supported platforms.